runner/ansible/roles/1.5.3/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '1.5.3'
+
+- name: "{{ test_id }}.check"
+  shell: |
+          # The third field of ntpq -p is the stratum and if it's 16
+          # the sync is disabled. Make sure it's not 16
+          if ! sudo ntpq -pn | grep "No association ID's returned" \
```

Comment:
Do we need to have `sudo` here?
I thought ansible would execute this already with root rights

Reply:
If it's all run as root, there is no need in sudo here of course.

Reply:
We cannot/should not depend on this being run as the root user.